values.yaml

```diff
@@ -188,6 +188,14 @@ server:
   # See https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   updateStrategyType: "OnDelete"
 
+  # Configure the logging verbosity for the Vault server.
+  # Supported log levels include: trace, debug, error, warn, info
```

Comment:
Can you put these in order?

Reply:
Shouldn't `error` be last?

Reply:
+1, in descending order of verbosity it should be trace, debug, info, warn, error

Reply:
Fixed.